src/util/fetch.ts

```diff
@@ -2,6 +2,9 @@ import { error } from 'console'
 // Exported cookie name to easily use the same name for the same cookie accross all application to avoid misprints
 import { getCookie , setCookie, deleteCookie, cookieName } from '../util/Authentication';
 
+//Export the server URL
+export const baseURL = window.location.hostname === 'localhost'? 'http://localhost:8000/api/v1/': 'https://shelf-share.onrender.com'
```

Comment:
shouldn't the render.com link have /api/v1 in it as well?

Reply:
@jpremo as far as I know we need to add it by ourselves, that's what I did when setup documentation and testing the prod URL in Postman.

Reply:
@jpremo I got confused first, but thanks for the comment.
@larasodjati, if we dont add it, it shows route dont exist so we need to add it to access like books or messages ...

Reply:
@Manizha-khorram true! Thats why we need to add it manually like what I said before, unless there's a setting to add it automatically during the deployment. @jpremo could you please approve this?